src/main/java/com/spring/studentmanagement/models/Address.java

```diff
@@ -0,0 +1,39 @@
+package com.spring.studentmanagement.models;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+/**
+ * Created at 23.04.2023 by Dan.
+ */
+@Getter
+@Setter
+@ToString
+@RequiredArgsConstructor
```

Comment:
Do we need @RequiredArgConstructor? YAGNI principle - You Are Not Gonna Need It. Do not write code unless you are using it